flask_project/campaign_manager/templates/campaign_detail.html

```diff
@@ -239,8 +243,19 @@ <h3 class="detail-campaign-name">{{ name }}</h3>
             }
 
             calculateProgress();
+            calculateContributors();
         });
 
+        function calculateContributors () {
+            var url = "/campaign_manager/campaign/" + uuid +"/total_contributors";
+            $('#contributors').html('<span class="grey-italic">Loading data...<span>');
```

Comment:
hi @ann26 
could you move this html directly to it's div?
`<span id="contributors"><span class="grey-italic">Loading data...<span></span>`

no need to add this because it is default for html.